### PR TITLE
added rockscontainer optimizations for all finished runs to aim reindex command

### PR DESCRIPTION
**Pull request by alberttorosyan**
by default `flush` + `flush_wal` will be applied to all `seqs` containers.
additionally compactions will be done for `meta` containers.

this can be turned off by running `aim reindex --repo <REPO_PATH> --finalize-only`, and effectively restore previous behavior of `aim reindex`

*NOTE this PR requires changes from `feature/run-wal-flush-on-container-readonly-open` to be merged into base.*

### CODE REVIEW

File: aim/cli/reindex/utils.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            click.secho(f'\t\'{run_hash}\'', fg='yellow')
+
+
+def run_flushes_and_compactions(repo: 'Repo', runs_to_skip: set):
+    meta_dbs_path = os.path.join(repo.path, 'meta', 'chunks')
+    seq_dbs_path = os.path.join(repo.path, 'seqs', 'chunks')
+    meta_dbs_names = set(os.listdir(meta_dbs_path)).difference(runs_to_skip)
+    seq_dbs_names = set(os.listdir(seq_dbs_path)).difference(runs_to_skip)
+
+    pool = ThreadPool(cpu_count())
```
Comment (mihran113) [inline aim/cli/reindex/utils.py:39]: `psutil.cpu_count(logical = False)` let's stick with this method to get cpu count, so we'll get the count of physical cores only

Comment (alberttorosyan) [reply aim/cli/reindex/utils.py:39]: 👌

**Comment by alberttorosyan** [comment]
resolves #963 : cli command to optimize existing repos

**Review by mihran113 — APPROVED** [review]
lgtm

**Pull request merged**